### PR TITLE
Upgrade rusty_v8 to 0.2.0

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

File: core/bindings.rs [hunk 1]
```diff
@@ -397,8 +398,9 @@ fn send(
 
   let control = match v8::Local::<v8::ArrayBufferView>::try_from(args.get(1)) {
     Ok(view) => {
-      let mut backing_store = view.buffer().unwrap().get_backing_store();
-      let backing_store_ptr = backing_store.data() as *mut _ as *mut u8;
+      let backing_store = view.buffer().unwrap().get_backing_store();
+      let backing_store_ptr =
+        unsafe { &mut *backing_store.get() } as *mut _ as *mut u8;
```
Comment (piscisaureus) [inline core/bindings.rs]: Remove the from_raw parts stuff below.
Backingstore derefs to slice already.

File: core/isolate.rs [hunk 2]
```diff
@@ -48,8 +48,9 @@ unsafe impl Send for PinnedBuf {}
 
 impl PinnedBuf {
   pub fn new(view: v8::Local<v8::ArrayBufferView>) -> Self {
-    let mut backing_store = view.buffer().unwrap().get_backing_store();
-    let backing_store_ptr = backing_store.data() as *mut _ as *mut u8;
+    let backing_store = view.buffer().unwrap().get_backing_store();
+    let backing_store_ptr =
+      unsafe { &mut *backing_store.get() } as *mut _ as *mut u8;
```
Comment (piscisaureus) [inline core/isolate.rs]: Ditto

File: core/bindings.rs (hunk 1 above)
Comment (ry) [reply core/bindings.rs]: done

File: core/isolate.rs (hunk 2 above)
Comment (ry) [reply core/isolate.rs]: done

**Comment by ry** [comment]
> There's a transmute in bytes_mut() that can also be removed if you have the time.

I don't know which transmute you mean. There don't seem to be any in core anymore.

It seems there's a problem with the windows build - can you repeat the failure?

**Pull request merged**